justfile

```diff
@@ -43,3 +43,47 @@ clean:
 # Generate all four flavors
 all: setup
     {{whiskers_cmd}} {{template_path}}
+
+# ---- uosc ----
+
+# Variables
+output_uosc := "themes-uosc"
+template_path_uosc := "uosc.tera"
+
+# Create the output directory
+# TODO remove when PR #5 is merged
+setup_uosc:
+    # Make the theme directories
+    mkdir -p {{output_uosc}}/latte
+    mkdir -p {{output_uosc}}/frappe
+    mkdir -p {{output_uosc}}/macchiato
+    mkdir -p {{output_uosc}}/mocha
+
+    # Build the directory structure for one of the themes
+    mkdir -p {{output_uosc}}/latte/rosewater/script-opts
+    mkdir -p {{output_uosc}}/latte/flamingo/script-opts
+    mkdir -p {{output_uosc}}/latte/pink/script-opts
+    mkdir -p {{output_uosc}}/latte/mauve/script-opts
+    mkdir -p {{output_uosc}}/latte/red/script-opts
+    mkdir -p {{output_uosc}}/latte/maroon/script-opts
+    mkdir -p {{output_uosc}}/latte/peach/script-opts
+    mkdir -p {{output_uosc}}/latte/yellow/script-opts
+    mkdir -p {{output_uosc}}/latte/green/script-opts
+    mkdir -p {{output_uosc}}/latte/teal/script-opts
+    mkdir -p {{output_uosc}}/latte/sky/script-opts
+    mkdir -p {{output_uosc}}/latte/sapphire/script-opts
+    mkdir -p {{output_uosc}}/latte/blue/script-opts
+    mkdir -p {{output_uosc}}/latte/lavender/script-opts
+
+    # Copy the directory structure for the other themes
+    cp -vr {{output_uosc}}/latte/* {{output_uosc}}/frappe
+    cp -vr {{output_uosc}}/latte/* {{output_uosc}}/macchiato
+    cp -vr {{output_uosc}}/latte/* {{output_uosc}}/mocha
+
+# Remove all files in the output directory
+clean_uosc:
+    rm -rfv {{output_uosc}}
+
+# Generate all four flavors
+all_uosc: setup_uosc
```

Comment:
It might be a good idea to have `clean_uosc` and `all_uosc` be dependencies of `clean` and `all`.

Reply:
Well, my idea was to keep the main theme separate.

Now, with a separate justfile in a subdirectory I added `just uosc/{clean,all}` in front. Idk if there is a cleaner way.

Reply:
Understandable. With the subdirectory and separate justfile, it should be alright how it currently is.